src/main/java/uk/gov/hmcts/reform/pip/data/management/service/SubscriptionManagementService.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.pip.data.management.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.WebClientException;
+import uk.gov.hmcts.reform.pip.data.management.models.external.Subscription;
+import uk.gov.hmcts.reform.pip.data.management.models.publication.Artefact;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+@Component
+public class SubscriptionManagementService {
+
+    @Value("${service-to-service.subscription-management}")
+    private final String url;
+
+    @Autowired
+    public SubscriptionManagementService(@Value("${service-to-service.subscription-management}") String url) {
```

Comment:
Not sure you need this @Autowired :) 

The @Value should be enough to cover this

Reply:
cheers, yep sorted.